components/match2/wikis/trackmania/match_group_input_custom.lua

```diff
@@ -173,8 +184,12 @@ function matchFunctions.getExtraData(match)
 		and opponent2.type == Opponent.team
 
 	match.extradata = {
+		lastgame = Variables.varDefault('last_game'),
```

Comment:
that (wiki) var is never set anywhere (no non deprecated template/module sets it)